activesupport/test/inflector_test.rb

```diff
@@ -592,7 +608,7 @@ def test_clear_with_default
     end
   end
 
-  %w(plurals singulars uncountables humans acronyms).each do |scope|
+  %i(plurals singulars uncountables humans).each do |scope|
```

Comment:
Is it correct that `acronyms` is deleted?
Also using symbols instead of strings doesn't seem necessary.

Reply:
I removed `acronyms` from this test case because it's now covered by this test:
https://github.com/rails/rails/pull/42475/files#diff-329546df3dce71c45102377d2f9eff2a8fdda92c739d67edd2044bd078938b56R472

And the change from string (`%w`) to symbol (`%i`) is because `#clear` is documented as expecting symbols (the implementation switches on symbols, not strings):
https://github.com/rails/rails/blob/051b0fcd29e927ec8adf7ac11d3a469645e1d8d6/activesupport/lib/active_support/inflector/inflections.rb#L224-L225

Reply:
Ah yes, because it would allow everything before. 👍 
Maybe we should raise if a provided option isn't supported.